aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/plugins/article_search.html

```diff
@@ -0,0 +1,33 @@
+{% load i18n sekizai_tags %}
+
+{% addtoblock "js" %}
+<script>
+(function($){
+    $('.aldryn-newsblog-article-search').find('form').on('submit', function(evt){
+        var $form = $(this);
+
+        evt.preventDefault();
+        $.ajax({
+            type: 'get',
+            url: $form.attr('action'),
+            data: $form.serialize()
+        }).always(function(data){
+            $form.siblings('.search-results').html(data);
+        });
+    });
+}(window.jQuery));
+</script>
+{% endaddtoblock %}
+
+<div class="aldryn aldryn-newsblog aldryn-newsblog-article-search">
+    <form class="form-inline" action="{{ query_url }}" method="POST">
```

Comment:
we have the guideline to always use lowercase in html: https://aldryn-boilerplate-bootstrap3.readthedocs.org/en/latest/html/guidelines.html